src/components/organisms/Navbar.tsx

```diff
@@ -75,6 +80,11 @@ export const Navbar = (connect as any)(mapStateToProps, mapDispatch)(
               </form>
             </div>
           )}
+          <Feature>
+            {({ features }) =>
+              isActive('nav-buttons', features) ? <NavButtons location={this.props.location} /> : <Noop />
```

Comment:
Just curious — what happens if you simply don't return anything (or, rather, null or undefined) rather than Noop?

Something like `isActive('nav-buttons', features) && <NavButtons location={this.props.location} />`

Reply:
It worked!